backend/src/baserow/contrib/automation/migrations/0001_squashed_0023_initial.py

```diff
@@ -0,0 +1,891 @@
+# Generated by Django 5.0.14 on 2025-11-13 16:29
+
+import django.db.migrations.operations.special
+import django.db.models.deletion
+from django.db import migrations, models
+
+import baserow.contrib.automation.nodes.models
+import baserow.core.fields
+import baserow.core.mixins
+
+
+# baserow.contrib.automation.migrations.0013_apply_previous_node_ids
+def apply_previous_node_ids_0013_forward(apps, schema_editor):
```

Comment:
Hey @picklepete quick question, do you know why django is keeping this data migrations? Could we remove them or do you think they are necessary? I don't see the point of migrating data for empty databases...